pages/apis/rest_api/clusters.md

```diff
@@ -89,3 +89,141 @@ curl "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{id}"
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>description</code></th><td>Description for the cluster.<br><em>Example:</em> <code>"A place for safely running our open source builds"</code>
+  <tr><th><code>emoji</code></th><td>Emoji for the cluster using the <a href="/docs/pipelines/emojis">emoji syntax</a>.<br><em>Example:</em> <code>"\:technologist\:"</code>
+  <tr><th><code>color</code></th><td>Color hex code for the cluster.<br><em>Example:</em> <code>"#FFE0F1"</code>
+</tbody>
+</table>
+
+Required scope: `write_clusters`
+
+Success response: `201 Created`
+
+Error responses:
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>422 Unprocessable Entity</code></th><td><code>{ "message": "Validation failed: ..." }</code></td></tr>
+</tbody>
+</table>
+
+## Update a cluster
+
+```bash
+curl -X PUT "https://api.buildkite.com/v2/organizations/{org.slug}/clusters" \
+  -H "Content-Type: application/json" \
+  -d '{ "name": "Hack the Planet" }'
+```
+
+```json
+{
+  "id": "42f1a7da-812d-4430-93d8-1cc7c33a6bcf",
+  "graphql_id": "Q2x1c3Rlci0tLTQyZjFhN2RhLTgxMmQtNDQzMC05M2Q4LTFjYzdjMzNhNmJjZg==",
+  "name": "Hack the Planet",
+  "description": "A place for safely running our open source builds",
+  "emoji": "\:technologist\:",
+  "color": "#FFE0F1",
+  "url": "http://api.buildkite.com/v2/organizations/acme-inc/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf",
+  "web_url": "http://buildkite.com/organizations/acme-inc/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf",
+  "created_at": "2023-05-03T04:17:55.867Z",
+  "created_by": {
+    "id": "3d3c3bf0-7d58-4afe-8fe7-b3017d5504de",
+    "graphql_id": "VXNlci0tLTNkM2MzYmYwLTdkNTgtNGFmZS04ZmU3LWIzMDE3ZDU1MDRkZQo=",
+    "name": "Sam Kim",
+    "email": "sam@example.com",
+    "avatar_url": "https://www.gravatar.com/avatar/example",
+    "created_at": "2013-08-29T10:10:03.000Z"
+  }
+}
+```
+
+[Request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>name</code></th><td>Name for the cluster.<br><em>Example:</em> <code>"Open Source"</code>
+  <tr><th><code>description</code></th><td>Description for the cluster.<br><em>Example:</em> <code>"A place for safely running our open source builds"</code>
+  <tr><th><code>emoji</code></th><td>Emoji for the cluster using the <a href="/docs/pipelines/emojis">emoji syntax</a>.<br><em>Example:</em> <code>"\:technologist\:"</code>
+  <tr><th><code>color</code></th><td>Color hex code for the cluster.<br><em>Example:</em> <code>"#FFE0F1"</code>
+</tbody>
+</table>
+
+Required scope: `write_clusters`
+
+Success response: `200 OK`
+
+Error responses:
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>422 Unprocessable Entity</code></th><td><code>{ "message": "Validation failed: ..." }</code></td></tr>
+</tbody>
+</table>
+
+## Delete a cluster
+
+Delete a cluster and any cluster queues and cluster tokens that belong to it.
```

Comment:
Do you think it's important to repeat "cluster" each time? I wonder if it might be easier to read as:

```suggestion
Delete a cluster and any queues and tokens that belong to it.
```

Reply:
Actually, maybe to simplify the "ands:"

```suggestion
Delete a cluster along with any queues and tokens that belong to it.
```

Reply:
That reads so much better! 🙇